### DISCUSSION TITLE
[Newbie] Need more install procedures using Docker

### DISCUSSION THREAD

**Discussion by rthien** [OpnForm/OpnForm#139] [Q&A]
Hi,

I am trying to help a NFP organization install this on their Synology NAS with Docker containers enabled. I tried the steps given but I was not able to make it work.

Yes, I don't have much experience with Docker and any help will be much appreciated. The NFP has been using Google Forms but with their new policies on storage, the NFP is limited by the amount of data they can collate now.

TIA!

**Comment by JhumanJ** [comment]
We had issues with our docker setup, but it should be resolved now.
